bookkeeper-server/src/test/java/org/apache/bookkeeper/client/BookKeeperAdminTest.java

```diff
@@ -230,4 +241,200 @@ public void testDecommissionForLedgersWithMultipleSegmentsAndNotWriteClosed() th
         }
         bkAdmin.close();
     }
+
+    @Test(timeout = 60000)
```

Comment:
Since we have a default timeout setting for all tests, it is suggested to remove `timeout` for individual test case.

Reply:
what is the default time? and what if we want more time?

Reply:
- default time is more than 60 seconds
- typically if a test is required more than 60 seconds, then we really need to think about how to improve the test. A lot of frakiness in current bookkeeper tests come from unnecessarily `Thread.sleep` or wait for something to trigger at a given time. we should try to avoid that pattern.

Reply:
removed timeouts